### PR TITLE
Fixing compose file

### PR DESCRIPTION
**Pull request by ruffsl**
* moving Dockerfile to squid folder 
* removing `link` in compose file to fix net `host` conflict

### CODE REVIEW

**Comment by tfoote** [comment]
The README needs to be updated for using compose instead of fig too.

**Comment by jpetazzo** [comment]
Thanks! Just one question, why did you change the example in the YAML file? (I mean, the extra env vars...?)

File: README.md [hunk 1]
```diff
@@ -38,15 +38,16 @@ iptables -t nat -D PREROUTING -p tcp --dport 80 -j REDIRECT --to 3129 -w
...
 which will setup the iptables rules for you. To use this you will need a
-local checkout of this repo and have `fig` and `docker` installed.
+local checkout of this repo and have `docker` and `compose` installed.
+
+> Run the following command in the same directory as the `docker-compose.yml` file:
 
-1. Run the following commands
 ```bash
-fig up -d squid && fig run tproxy
+docker-compose up
```
Comment (thomasleveil) [inline README.md]: shouldn't that be `docker-compose up && docker-compose run tproxy`?

Comment (ruffsl) [inline README.md]: Nope, you shouldn't need to run each service explicitly.
Using just the up command with start every service defined in the .yml if needed.
Is it not working for you?

**Comment by ruffsl** [comment]
I added back the extra env vars, though I'm not sure what they do.
Mind informing me what they are, and checking if the yaml string format works?
@jpetazzo

File: README.md (hunk 1 above)
Comment (tfoote) [inline README.md]: If I remember correctly the two separate commands were to work around what appeared to be a bug in fig or docker that I was never able to fully chase down: https://github.com/tfoote/squid-in-a-can/issues/

Comment (ruffsl) [inline README.md]: As with Docker v1.8 and Docker compose v1.4.0, `docker-compose up` seems to be working for me.

**Comment by jpetazzo** [comment]
Tested locally, all seems good.

The env vars are there to give examples for the most common Squid parameters.

If my memory serves me well:

- DISK_CACHE_SIZE as the name implies, control total disk usage of Squid
- MAX_CACHE_OBJECT controls size of biggest object (bigger objects are not cached)
- SQUID_DIRECTIVES_ONLY and SQUID_DIRECTIVES are used by the custom Python script (I didn't write it so I don't remember 100% what they do!)

**Pull request merged**